datafusion/core/tests/sqllogictests/test_files/select.slt

```diff
@@ -21,9 +21,8 @@
 ##########
 
 # all where empty
-query II
+statement ok
```

Comment:
I don't understand the need for any of the changes in this file

Reply:
I think I just ran `--complete` and this is what it produced :sweat_smile: